### ISSUE TITLE
Improved accessibility for screen readers

### ISSUE THREAD

**Issue by insomnious** [Nexus-Mods/NexusMods.App#2933]
# Bug Report

## Summary

Received via _Nexus Mods App Feedback Google Form_

Screen readers aren't able to read some buttons and controls effectively

> So, I am blind and use talking software, typically known as a screen reader. The one I use is called NVDA, Non-Visual Desktop Access. This allows me to use my computer as efficiently as my sighted counterparts. When I open the Nexus Mods apps for windows. Many buttons aren't labeled so that NVDA can read them. Some are labeled, but, in an odd way such as. Avalonia.Controls.Grid, or, Avalonia.Controls.DockPanel. Some are labeled correctly such as, Click to give feedback and help us improve., or, Open Discord, or even, Open Forums at Nexus Mods. But, as far as being able to see the mods I have installed or anything else the app may allow me to do. With out visual help, I am unable to do these things. Would you be willing to look up how NVDA works and try to make the Nexus Mods app more accessible to the blind? I mean, I could use the website as I've always done, but, I would love to be able to use the app as, if I understand it correctly, I can actually manage mods directly from the app and that is something I've been wanting to be able to do ever sense I started playing Stardew Valley and I learned you could add mods. The game is made accessible do to the Stardew Access mod. I love it very much!!! Thanks so much for your help and understanding. If you need further help in understanding screen readers, or, you would like help in testing for accessibility. Please let me know. I'd be glad to help make your app more accessible and usable by more people.

## Steps to reproduce

- Install NVDA or Accessibility Insights
- Focus on the app
- Some controls have none to little accessibility data associated with them, some controls have too much and shouldn't be focusable

## What is the expected behaviour?

- The app should be easy to navigate for accessibility apps and screen readers

## Other information

I've started investigating into what Avalonia is doing by default and what our custom controls are doing. I'll create specific tickets to retrofit the app as a first phase so that the screen readers 'read' the app better, I've already found some easy wins. We will also need to plan how to fit accessibility into the QA process better.

 When viewing the App with Accessibility Insights, specifically our DebugControls page, 'button' is our `StandardButton` control.

![Image](https://github.com/user-attachments/assets/583ac4c2-7748-42d4-b33a-6d3410a8482f)

| Before | After |
|--------|--------|
| ![Image](https://github.com/user-attachments/assets/ed221c9f-7054-407e-8f29-c0ccfbbdadec) | ![Image](https://github.com/user-attachments/assets/1f1f96ea-d254-4884-b386-8ec5835b4aed) |
 
### Software used

- [Accessibility Insights for Windows](https://accessibilityinsights.io/docs/windows/overview/)
- [NVDA](https://www.nvaccess.org/about-nv-access/)
